### DISCUSSION TITLE
codespace bug

### DISCUSSION THREAD

**Discussion by A5MA5T3R** [community/community#54221] [Codespaces]
### Select Topic Area

General

### Body

Hello, this year I'm doing a project on github and I had a big problem yesterday. I'm new to github, so I don't know how to solve my problem. 
I am coding on my codespace  in my browser, and after I executed a new part of my program yesterday, my github project just dont work anymore. 
The problem is that I wrote a lot of code since my last commit, so I can t just open a new codespace and retrieve the changes. I tried to open my codespace on vscode or another browser and I have the same problem. I tried the option "export changes to a branch"
![image](https://user-images.githubusercontent.com/99291310/235598558-5c9a5ad4-d093-42f6-8de7-f43bb622aae2.png)
but it didnt work, and apparently it didn't work.
I hope you'll be able to help me.

**Comment by A5MA5T3R** [comment]
here is what happend when I wanna charge my codespace : 
![image](https://user-images.githubusercontent.com/99291310/235599566-2a30bd3e-b2c2-4627-adb0-49173c62ed3e.png)

**Reply by A5MA5T3R** [reply]
I got this as well 
![image](https://user-images.githubusercontent.com/99291310/235601891-6a1fe950-93a8-40e2-bdfb-c46dca77aa19.png)

**Comment by dmgardiner25** [comment]
Hi @A5MA5T3R, thanks for opening this discussion. I took a look at why exporting your changes failed and it looks like you are exceeding the size limit that GitHub has in place when committing either due to having too large of a file or having too many files in the commit.

Are you still able to connect to your codespace through VS Code? If not, one option is to install the [GitHub CLI](https://cli.github.com/) locally on your computer which should allow you to connect to it via your shell. If that also doesn't work, we can find some alternative options to get you your changes.
